### PR TITLE
add  array type

### LINKED ISSUE #6: add `explain` array property

### PR DESCRIPTION
**Pull request by pftom**
Close #6

### CODE REVIEW

File: src/components/utils/common.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const isArray = (elem) => Array.isArray(elem);
+
+export {
+  isArray,
+}
```
Comment (mRcfps) [inline src/components/utils/common.js]: Err, it seems there is no newline at the end of file. Please add one.

File: src/components/utils/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from './common';
```
Comment (mRcfps) [inline src/components/utils/index.js]: New line please.

File: src/components/Content.js [hunk 3]
```diff
@@ -20,7 +21,14 @@ export default class Content extends Component {
             {this.props.viewType}
           </Button>
         </div>
-        <p>{explain}</p>
+        {
+          isArray(explain)
```
Comment (mRcfps) [inline src/components/Content.js]: Why not use `Array.isArray` here directly? I'm not familliar with jsx.

File: src/components/utils/index.js (hunk 2 above)
Comment (pftom) [reply src/components/utils/index.js]: I will add a .editorconfig later

File: src/components/Content.js (hunk 3 above)
Comment (pftom) [reply src/components/Content.js]: I want write a collection of javascript utility function. because, later, We will have a lot of reuse logic.

Comment (pftom) [reply src/components/Content.js]: Also, `isArray` will more be more complex, not like now

**Pull request merged**